examples/fork/fork.md

```diff
@@ -0,0 +1,7 @@
+# Fork
+
+The fork statement starts with one or more named workers, which run in parallel with each other, each in its own new strand. Variables and parameters in the scope of the fork statement remain within the scope of the workers. Message passing can be done only between the workers created within the fork statement and cannot be done with the function's default worker and its named workers. Unlike named workers, the `fork` keyword can be used to define workers anywhere within the function body.
```

Comment:
```suggestion
The fork statement starts one or more named workers, which run in parallel with each other, each in its own new strand. Variables and parameters in the scope of the fork statement remain within the scope of the workers. Message passing can be done only between the workers created within the fork statement and cannot be done with the function's default worker and its named workers. Unlike named workers outside a `fork` statement, with a `fork` statement, workers can be defined anywhere within the function body (e.g., within a conditional block).
```

> Variables and parameters in the scope of the fork statement remain within the scope of the workers

may make sense to include an example

Reply:
Updated [2424cde](https://github.com/ballerina-platform/ballerina-distribution/pull/5045/commits/2424cde4faf9430cb7e1d4f59fe7ca7aea023876)